src/pedalintandem.py

```diff
@@ -0,0 +1,114 @@
+import json
+from datetime import datetime
+from curl_cffi import requests
+from common.tz import IST
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://www.pedalintandem.com"
+
+def _date(date_str):
+    return datetime.fromisoformat(date_str).astimezone(IST).isoformat()
+
+
+def fetch_events_links(session):
+	res = session.get(f"{BASE_URL}/experiences", impersonate="chrome")
+	soup = BeautifulSoup(res.text, 'html.parser')
+	event_divs = soup.find_all('div', class_ = 'single-experience')
+
+	# Fetch events from other pages.
+	load_more = soup.find('div', class_ = 'products-loadmore')
+	while load_more != None:
+		url = load_more.find('a').get('href')
+		
+		new_page = session.get(f"{BASE_URL}{url}")
+		soup = BeautifulSoup(new_page.text, 'html.parser')
+		
+		new_events = soup.find_all('div', class_ = 'single-experience')
+		event_divs += new_events
+
+		load_more = soup.find('div', class_ = 'products-loadmore')
+
+	event_links = []
+	for event_div in event_divs:
```

Comment:
unsure about the need for event_divs. Just iterate over anchor tags instead.

Reply:
Will check this and update it.